### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse - New Features_Edit Configuration popup_Keyboard: The dropdowns “Job upload storage, Advanced configuration, Before launch” present under Upload path heading are not accessible using keyboard.

### ISSUE THREAD

**Issue by v-lasiva** [microsoft/azure-tools-for-java#3596]
[Check out Accessibility Insights! ](url)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_JavaToolingforIntelliJ/Eclipse–New Feature; #MAS2.1.1; #Keyboard; #C+E-H; #JavatoolingforIntelliJ/Eclipse;

**Environment Details:**
URL: https://www.jetbrains.com/idea/download/#section=windows
Version: 2019.2.2
Build: 192.6603.28
MAS Violated: MAS 2.1.1

**NOTE:
1.Same issue is observed in “Spark Console” scenario also.
(Run --> Edit Configurations-->Apache Spark on HDInsight > [Spark on HDInsight] myApp)
Here also, “Before Launch” dropdown is not accessible using keyboard.
2.Same issue is observed in “Create Spark Scala” Scenario also.
(New-->Project and press enter
Select Azure Spark/HDInsight > Spark Project with Samples (Scala) and press Next.)**

Here also, “More settings” dropdown is not accessible using keyboard.

**Repro Steps:**
1.Download IntelliJ application from the URL: https://www.jetbrains.com/idea/download/#section=windows .
2.Launch IntelliJ IDEA.
3.Tab till Create New Project button and press enter
4.Tab till Azure Spark/HDInsight-->Spark Project Scala
5.Tab till Build Tools-->Maven
6.Tab till Next button and press enter
7.Tab till finish button and press enter
8.Navigate to project name-->src-->main-->scala-->project name and open the context menu using keyboard key
9.Tab till Submit Spark Application and press enter.
10.Tab till “Referenced files” field.
11.Verify whether the dropdowns “Job upload storage, Advanced configuration, Before launch” present under Upload path heading are accessible using keyboard or not.

**Actual Result:**
The dropdowns “Job upload storage, Advanced configuration, Before launch” present under Upload path heading are not accessible using keyboard.

**Expected Result:**
The dropdowns “Job upload storage, Advanced configuration, Before launch” present under Upload path heading should be accessible using keyboard.

**User Impact:**
Keyboard users will face difficulty in accessing and knowing the information present in those dropdowns if they are not accessible using keyboard.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

[Bug_Dropdowns not accessible using keyboard in edit configuration popup.pptx](https://github.com/microsoft/azure-tools-for-java/files/3641819/Bug_Dropdowns.not.accessible.using.keyboard.in.edit.configuration.popup.pptx)

**Comment by t-rufang** [comment]
Note:When focus is at `Referenced Files`, press tab, focus goes to `SparkJobRun`. But we expect focus should be at `Job Upload Storage` part.
![image](https://user-images.githubusercontent.com/32627233/65597998-05328f00-dfcd-11e9-802a-6d59713de176.png)

**Comment by t-rufang** [comment]
The job upload storage part is a hideable panel, which extends class `com.intellij.ui.HideableTitledPanel` and this class is supported by IntelliJ. Please find the same issue in the `Before launch: Build, Activate tool window` hideable panel.
![image](https://user-images.githubusercontent.com/32627233/67269672-bacc0180-f4e9-11e9-96ea-7be906b0e9de.png)

**Comment by t-rufang** [comment]
- Importance of experience: Minimal, keyboard only users can't navigate to the "Before launch" panel. This section indicate what need to be done before we run/debug a job based on this configuration. For example, we can add a "build project" task in this section. 
![image](https://user-images.githubusercontent.com/32627233/69412720-6b782b80-0d4a-11ea-99de-c305b5c5ac4f.png)

- Quality of experience: Annoying, this "Before launch" panel is hardly used when user want to submit a Spark job locally or remotely. And the actions in "Before launch" section can be mannually executed if necessary.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue.
- [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by v-lasiva** [comment]
As per discussion with SME, Closing the issue.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749699
